### ISSUE TITLE
Path with space causes images not to load

### ISSUE THREAD

**Issue by kenjibailly** [kbravh/obsidian-tweet-to-markdown#10]
Images won't load when there's a space in the path like this:

```
![mindglowingart](Obsidian Vault/Archive/Assets/mindglowingart-1495795363961319425.jpg)
```

Now when replacing the space with the code char "%20" it works.
Got me thinking to change the Image Location path in the settings to:
```
Obsidian%20Vault/Archive/Assets/
```

Unfortunately this doesn't work everywhere as when it downloads an image it will create a new folder "Obsidian%20Vault" like in screenshot below:
![image](https://user-images.githubusercontent.com/9883910/156866042-33b9362a-3369-47fe-b132-728cdb521f31.png)

An easy fix would be to just use this kind of format instead:

```
![[Obsidian Vault/Archive/Assets/mindglowingart-1495795363961319425.jpg]]
```

OR

To fix the %20 problem

**Comment by kbravh** [comment]
Interesting bug, thanks for catching that! Let me get to work on a fix.

**Comment by kenjibailly** [comment]
> Interesting bug, thanks for catching that! Let me get to work on a fix.

You're faster than paid support! :o
Thank you! <3

Edit: Not sure why I wanted to download the images, thought that if not selected images wouldn't show up, but they do. Probably not going to use the download images. Sorry for the trouble!

**Comment by kbravh** [comment]
Haha, no worries! I'm always happy when people find bugs because I love that they're using the plugin.

I've just published a fix in version 2.2.0. I added a new option to the plugin settings that allow you to choose between markdown image embeds vs obsidian image embeds. I've also added proper handling for URI encoded spaces like %20.
